jira/resources.py

```diff
@@ -763,14 +763,46 @@ def __init__(
             self._parse_raw(raw)
         self.raw: Dict[str, Any] = cast(Dict[str, Any], self.raw)
 
-    def update(self, fields=None, async_=None, jira=None, body="", visibility=None):
-        """Update a comment"""
-        data = {}
+    # mypy complains about the signature as the order of the parameters is different.
+    # If we follow mypy here we will break backwards compatibility with version <= 3.2 which is probably not advised
+
+    def update(  # type: ignore[override]
+        self,
```

Comment:
would be better if we could keep this comment closer to its use, and I don't think all of this is correct

```suggestion

    def update(  # type: ignore[override] # We've added new parameters and order of parameters is different
        self,
```

maybe we could even do, if the formatter is happy with it

```suggestion

    def update(  # type: ignore[override]
        # We've added new parameters and order of parameters is different
        self,
```